### ISSUE TITLE
Опечатка на картинке в общем весе и плотности монораствора

### ISSUE THREAD

**Issue by niksosov** [siv237/HPG#14]
https://github.com/WEGA-project/wega-hpg/wiki/Raschet#изготовление-монораствора
На картинке при добавлении 150г соли в 1500мл воды получился вес 1800г и плотность 1,2г/мл (верные значения должны быть 1650г и 1,1г/мл)
Далее по тексту эта ошибка была тоже перенесена в раздел https://github.com/WEGA-project/wega-hpg/wiki/Raschet#плотность
итоговый расчет `50 мл * 1.2 г/мл = 60 грамм` неверный, правильный расчет: `50 мл * 1.1 г/мл = 55 грамм`

**Comment by siv237** [comment]
это не опечатка, а пример с удобными для примера циферкаиюми просто передающими суть процесса

**Comment by niksosov** [comment]
согласен, но на картинках опять же берется 1500 мл воды и 150 г соли, те 1500 г + 150 г = 1650 г
но далее почему-то это превращается в вес 1800 г, что может ввести в заблуждение, те ранее полученные 1650 г ≠ 1800 г
